recipes/timml/meta.yaml

```diff
@@ -0,0 +1,43 @@
+# Jinja variables help maintain the recipe as you'll update the version only here
+{% set name = "timml" %}
+{% set version = "5.0.0" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
+  git_rev: v{{ version }}
+  git_url: https://github.com/mbakker7/timml.git
+
+build:
+  number: 0
+  skip: True  # [py<34]
+  string: py36_0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - scipy
+    - numpy x.x
+    - matplotlib >=1.4
+
+  run:
+   - python
+   - scipy
+   - numpy x.x
+   - matplotlib >=1.4
+   
+test:
+  imports:
+  - timml
+  
+about:
+  home: https://github.com/mbakker7/timml
+  license: MIT
```

Comment:
Can you add the `license_file`, as well as `dev_url: https://github.com/mbakker7/timml`